spec/requests/create_work_spec.rb

```diff
@@ -100,6 +100,25 @@
         end
       end
 
+      context 'with a valid work_type and not enough valid required subtypes' do
+        it 'redirects to dashboard with an informative flash message' do
+          get "/collections/#{collection.id}/works/new?work_type=mixed+material&subtype%5B%5D=Data"
+          expect(response).to redirect_to(dashboard_path)
+          follow_redirect!
+          expect(response).to have_http_status(:ok)
+          expect(response.body).to include 'Invalid subtype value for work_type &#39;mixed material&#39;: Data'
```

Comment:
Is this something that is useful to a user? It seems like they would be stuck at this point. I would be hesitant to create a test that says "the application must behave this way", unless this is the way the PO has requested it to behave.

Reply:
Maybe not, but this batch of tests also serves as coverage of the WorkSubtypeValidator which is not directly tested in its own set of unit tests.

Reply:
Can this move to a unit test on WorkSubtypeValidator then?  I want to avoid this being something that confuses us in the future.

Reply:
@jcoyne I'm happy to put up a small, new, follow-up PR that changes how we test the subtype validator, sure. Here's what I'm thinking re: scope---let me know if you agree or have any tweaks:

We should have a new work subtype validator spec that adds unit tests for the validator (to test all the validation-specific behaviors), and the validation-related tests in this requests spec can be reducde to one or two cases that test that the work subtype validator is wired up to the request without needing to test every validation behavior.

What do you think?

Reply:
I'd rather not have any tests with regard to how the request should behave if it's invalid without talking to @astridu and @amyehodge about what their expectations are in this case.  I think if we have this path, the user should have a way out of this dead end, or we should just let it throw an error.

Reply:
At a minimum drop a comment on this test that says "We have no requirement for this UX, but we want to show that it only accepts valid inputs"

Reply:
@jcoyne OK. We already have this behavior in the app. We can strip it out, of course. I'm reluctant to initiate making changes to that behavior this late in the work cycle, and would like to focus on test strategy. Instead of dropping a comment on this test, how about I do what I proposed above (with your delta of removing all request specs related to validation since not supported by design or P.O. requirements) in a follow-up PR?